data/webbrowser.yaml

```diff
@@ -78,6 +78,8 @@ sources:
     - '%%users.localappdata%%\Chromium\User Data\*\Cache\*'
     - '%%users.localappdata%%\Chromium\User Data\*\Media Cache\*'
     - '%%users.localappdata%%\Chromium\User Data\*\GPUCache\*'
+    - '%%users.localappdata%%\Microsoft\Edge\User Data\*\Cache\*'
```

Comment:
I'm wondering if it is more logical from a usage perspective to put these under a Microsoft Edge definition instead?

Reply:
this can be solved with an alias